### ISSUE TITLE
[Suggestion] [Block Request] Steeper stairs / ladders

### ISSUE THREAD

**Issue by GadyZ** [tsunamayo/Skywanderers#39]
[Reference from the expanse](https://cdn.discordapp.com/attachments/271723356795961345/289586685237264384/unknown.png)

**Comment by tsunamayo** [comment]
Yes indeed, I need to give some stairs with the new stretch tech it should looks great. Thanks

**Comment by GadyZ** [comment]
That was three years ago, François, hahaha
But still glad to hear you took the suggestion to heart!

**Comment by tsunamayo** [comment]
Haha indeed, I am doing a big cleaning of my huge case backlog! I hope you are doing good!
